### PR TITLE
Add Windows support

### LINKED ISSUE #3: Windows build
I saw that @oblute already tried to build for windows in conda-forge/staged-recipes#11195, and that this is apparently not considered supported upstream, but I wanted to add this issue to track it at least.

### PR DESCRIPTION
**Pull request by vnlitvinov**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

* I have re-renrendered the feedstock
* I have also added running upstream tests
* Closes #3.

cc @h-vetinari

### CODE REVIEW

File: recipe/bld.bat [hunk 1]
```diff
@@ -0,0 +1,26 @@
+pushd "%SRC_DIR%" || exit /b
+
+rem Inline "installzlib.bat" and modify so it can be executed without MSYS
+curl -O https://zlib.net/zlib1211.zip || exit /b
```
Comment (h-vetinari) [inline recipe/bld.bat]: Can we add this as another win-only source with an expected hash (and target folder) under the `source:` section of `meta.yaml`?

**Review by h-vetinari — COMMENTED** [review]
Thanks a lot for tackling this! Haven't had time to check `bld.bat` in depth, but a first comment.

File: recipe/bld.bat (hunk 1 above)
Comment (vnlitvinov) [reply recipe/bld.bat]: Nice thought, this would also allow removing the dependency on `curl` and `unzip` I guess... I'll try, thx!

Comment (vnlitvinov) [reply recipe/bld.bat]: Fixed in 6c9cb73

File: recipe/meta.yaml [hunk 2]
```diff
@@ -6,8 +6,11 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/openai/atari-py/archive/{{ version }}.zip
-  sha256: 2ab97bc59f196f91d1b2227ac1cece3ef3fda996d9acf7a8c93e866d4bdc9f3f
+  - url: https://github.com/openai/atari-py/archive/{{ version }}.zip
+    sha256: 2ab97bc59f196f91d1b2227ac1cece3ef3fda996d9acf7a8c93e866d4bdc9f3f
+  - url: https://zlib.net/zlib1211.zip  # [win]
+    sha256: d7510a8ee1918b7d0cad197a089c0a2cd4d6df05fee22389f67f115e738b178d  # [win]
+    folder: atari_py/ale_interface/src/zlib  # [win]
```
Comment (h-vetinari) [inline recipe/meta.yaml:13]: My personal preference is to align selectors, but just as a comment, not a request.
```suggestion
  - url: https://zlib.net/zlib1211.zip                                        # [win]
    sha256: d7510a8ee1918b7d0cad197a089c0a2cd4d6df05fee22389f67f115e738b178d  # [win]
    folder: atari_py/ale_interface/src/zlib                                   # [win]
```

**Review by h-vetinari — APPROVED** [review]
Formatting nit still stands, but not in the way of approval 🙃

**Comment by vnlitvinov** [comment]
@conda-forge/atari_py anything needed from me to get this reviewed?

**Comment by oblute** [comment]
LGTM, I'll merge. Thanks @vnlitvinov!

**Pull request merged**